common/workunit/workunit.cpp

```diff
@@ -6949,6 +6949,7 @@ wuTokenStates verifyWorkunitDAToken(const char * distributedAccessToken)
     case WUStateBlocked:
     case WUStateAborting:
     case WUStateUploadingFiles:
+    case WUStateSubmitted://eclserver
```

Comment:
@RussWhitehead why should some of the Workunit states be treated a valid, some of them not. And what does the //eclserver mean?